controlplane/pkg/nsmd/nsmd_crossconnect_client.go

```diff
@@ -179,7 +179,7 @@ func (client *NsmMonitorCrossConnectClient) dataplaneCrossConnectMonitor(datapla
 						client.xconManager.UpdateClientConnectionSrcStateDown(clientConnection)
 					}
 					if dst := xcon.GetLocalDestination(); dst != nil && dst.State == local_connection.State_DOWN {
-						client.xconManager.UpdateClientConnectionDstStateDown(clientConnection)
+						client.xconManager.UpdateClientConnectionDstStateDown(clientConnection, false)
```

Comment:
`client.xconManager` is a reference, it will panic if it'll be `nil`

Reply:
It could not be nil by design.